### DISCUSSION TITLE
Money game

### DISCUSSION THREAD

**Discussion by tk3369** [Humans-of-Julia/HoJBot.jl#4] [Ideas]
We used to have Unbelivabot but killed it due to moderation issues. One of the more "popular" function was an economy system where members have the ability to work (virtually) and make money.  People can also transfer money between themselves. Then you can use the money to buy virtual pets (e.g. pigeon or horse) and later get into a game of racing pets.

It would be nice to have a game like that here.

**Comment by erwingame** [comment]
Good idea
